helm/botkube/e2e-test-values.yaml

```diff
@@ -103,14 +130,18 @@ executors:
 settings:
   clusterName: sample
   upgradeNotifier: false
+  log:
+    level: debug
+    disableColors: false
 
 extraAnnotations:
   botkube.io/disable: "true"
 
-analytics:
```

Comment:
By default, analytics will be enabled for e2e tests? Wouldn't this corrupt the segment io reports?

Reply:
This was an extra field that was removed. 

Analytics is disabled above: https://github.com/kubeshop/botkube/pull/723/files/304c022529ce66078abf56ebb108c282c3630aa6#diff-e4ca49997c968583f3e7664b5a8bd8346165fffd0fae1abc1bcf97815af8696dR1